cli/tests/test_log_command.rs

```diff
@@ -1167,16 +1167,16 @@ fn test_graph_styles() {
     @     merge
     |\
     | \
-    |  o  side branch
```

Comment:
Yeah, this is the ascii breakage I mentioned.

Reply:
The default node template is defined in `settings.rs` for this reason.